dis_snek/models/discord_objects/guild.py

```diff
@@ -295,6 +295,18 @@ def my_role(self) -> Optional["Role"]:
                 return role
         return None
 
+    async def get_member(self, member_id: "Snowflake_Type") -> Optional["Member"]:
+        """
+        Return the Member with the given discord ID
+
+        Args:
+            member_id: The ID of the member:
+        Returns:
+            Member object or None
+        """
+
+        return await self._client.get_member(user_id=member_id, guild_id=self.id)
```

Comment:
IMO this should be consistent with `get_owner` method (as in, `return await self._client.cache.get_member(self.id, member_id)` )

Reply:
Sure, makes sense